aws/lambda-promtail/locals.tf

```diff
@@ -0,0 +1 @@
+data "aws_region" "current" {}
```

Comment:
More of a preference thing, but could we include a newline at the end of the document?